### ISSUE TITLE
Can no longer open Unc0ver app v6.0 or 5.3.1

### ISSUE THREAD

**Issue by justaplane** [pwn20wndstuff/Undecimus#2224]
On IOS 13.3 (missed the 14.3 window)
iPhone 11 Pro
Using Apple Developer Account & Cydia Impactor for signing

Steps to Reproduce:

1. I originally restarted phone
2.  Opened v 5.3.1 which was still on my phone and went to re-jailbreak
3.  After the popup ad for buying the new case, I received an error stating Jailbreak Completed, but then said there was an error extracting / installing dpkg. 
4. I deleted the Unc0ver app from the phone
5. I saw there was a new version, 6.0 available so i downloaded it
6. Installed via Cydia Impactor v 0.9.52 with legitimate Developer account, and app-specific password
7. Tried to open Unc0ver and it immediately crashes (will not open or let me in the app). 
8. Removed Unc0ver again
9. Tried to re-install older 5.3.1 version
10. Same issue as step 7.
11. Repeated steps 6 - 8 over and over again with same result.

End Result - Can not get Unc0ver app to open any longer.

**Comment by tenwx** [comment]
Same situation, IOS 12.4, IPhone 6.

**Comment by tenwx** [comment]
> Same situation, IOS 12.4, IPhone 6.

Solve the problem after changing to Sideloadly.

**Comment by justaplane** [comment]
Update - it appears the issue is caused when there is an "old" .ipa file or conflict of .ipa files in the file system. The only way I was able to get it resolved, was to actually switch to altStore, which allowed me to see that i had 2 version of the unc0ver.ipa sitting on the phone. v5.3.1 and version 6.0. I could not see these files by browsing through the files. Altstore let me delete both and i could start from fresh. I ended up doing a complete factory restore and then a rootFS befroe starting again with v6.0.

Long story short, it seems that even if you delete unc0ver from your iPhone, there is a just chance the files and .ipa remain which is what is causing the issue.

**Comment by Tweaker177** [comment]
I had this same problem after updating to the latest version. I had to downgrade back to the version I had previously to get it to work. Neither version works in a jailbroken state though.

Was it an extra ipa file stored somewhere, or a provisional profile that needed deleted?

I made like 10 app specific passwords in one night troubleshooting it lol and would like to only make one when trying again.  I spent hours pulling my hair out trying to update UO versions and ended up just reinstalling the same one. 🙈

I had noticed a ton of CommCenter errors, locationd errors, problems loading user defaults, and tons of notifications-related errors when monitoring the console log in XCode. I’m on 14.2 iPhone 12 Pro Max, back on UO 6.0.1 after 6.1.0 wouldn’t work for me.  Hopefully the new version fixes a lot of that. I don’t think it’s tweak related.

**Comment by bubbashatz** [comment]
https://youtu.be/jDMCzVhm4yI
Upgrade to 14.3 now window closes on 4/26/21

**Comment by justaplane** [comment]
> I had this same problem after updating to the latest version. I had to downgrade back to the version I had previously to get it to work. Neither version works in a jailbroken state though.
> 
> Was it an extra ipa file stored somewhere, or a provisional profile that needed deleted?
> 
> I made like 10 app specific passwords in one night troubleshooting it lol and would like to only make one when trying again. I spent hours pulling my hair out trying to update UO versions and ended up just reinstalling the same one. 🙈
> 
> I had noticed a ton of CommCenter errors, locationd errors, problems loading user defaults, and tons of notifications-related errors when monitoring the console log in XCode. I’m on 14.2 iPhone 12 Pro Max, back on UO 6.0.1 after 6.1.0 wouldn’t work for me. Hopefully the new version fixes a lot of that. I don’t think it’s tweak related.

It was an extra .IPA. Almost as if it got lost, and was never properly deleted or was corrupted somehow, because it didn't show as "installed app". I found it by switching temporarily to alstore, because it has a built in file browser once installed on the iphone. That's how i discovered it. Once I found it, I could delete it. I switched back to my normal cydia impactor since I have my own dev account and everything is working.

**Comment by justaplane** [comment]
> https://youtu.be/jDMCzVhm4yI
> Upgrade to 14.3 now window closes on 4/26/21

WOW! This does actually work!!! I was able to upgrade to 14.3 last night!
